src/store/user.ts

```diff
@@ -4,7 +4,7 @@ import useAuthStore from "./auth";
 
 const ENDPOINT = import.meta.env.VITE_API_ENDPOINT;
 
-interface User {
+export interface User {
```

Comment:
This file is supposed to export only useAuthStore. If we want to export types, we should create model folder